plugins/utils/startify/options.nix

```diff
@@ -307,59 +307,63 @@ with lib;
-  custom_indices =
-    helpers.defaultNullOpts.mkNullable (with helpers.nixvimTypes; maybeRaw (listOf str)) "[]"
-      ''
-        Use any list of strings as indices instead of increasing numbers. If there are more startify
-        entries than actual items in the custom list, the remaining entries will be filled using the
-        default numbering scheme starting from 0.
-
-        Thus you can create your own indexing scheme that fits your keyboard layout.
-        You don't want to leave the home row, do you?!
-
-        Example:
-        ```nix
-          ["f" "g" "h"]
-        ```
-      '';
+  custom_indices = helpers.defaultNullOpts.mkListOf' {
+    type = types.str;
+    pluginDefault = [ ];
+    description = ''
+      Use any list of strings as indices instead of increasing numbers. If there are more startify
+      entries than actual items in the custom list, the remaining entries will be filled using the
+      default numbering scheme starting from 0.
 
-  custom_header =
-    helpers.mkNullOrOption (with helpers.nixvimTypes; maybeRaw (listOf (maybeRaw str)))
-      ''
-        Define your own header.
+      Thus you can create your own indexing scheme that fits your keyboard layout.
+      You don't want to leave the home row, do you?!
+    '';
+    example = [
+      "f"
+      "g"
+      "h"
+    ];
+  };
 
-        This option takes a `list of strings`, whereas each string will be put on its own line.
-        If it is a simple `string`, it should evaluate to a list of strings.
+  custom_header = helpers.defaultNullOpts.mkListOf' {
+    type = types.str;
+    description = ''
+      Define your own header.
 
-        Example:
-        ```nix
-          [
-            ""
-            "     ███╗   ██╗██╗██╗  ██╗██╗   ██╗██╗███╗   ███╗"
-            "     ████╗  ██║██║╚██╗██╔╝██║   ██║██║████╗ ████║"
-            "     ██╔██╗ ██║██║ ╚███╔╝ ██║   ██║██║██╔████╔██║"
-            "     ██║╚██╗██║██║ ██╔██╗ ╚██╗ ██╔╝██║██║╚██╔╝██║"
-            "     ██║ ╚████║██║██╔╝ ██╗ ╚████╔╝ ██║██║ ╚═╝ ██║"
-            "     ╚═╝  ╚═══╝╚═╝╚═╝  ╚═╝  ╚═══╝  ╚═╝╚═╝     ╚═╝"
-          ]
-        ```
-      '';
+      This option takes a `list of strings`, whereas each string will be put on its own line.
+      If it is a simple `string`, it should evaluate to a list of strings.
+    '';
+    example = [
+      ""
+      "     ███╗   ██╗██╗██╗  ██╗██╗   ██╗██╗███╗   ███╗"
+      "     ████╗  ██║██║╚██╗██╔╝██║   ██║██║████╗ ████║"
+      "     ██╔██╗ ██║██║ ╚███╔╝ ██║   ██║██║██╔████╔██║"
+      "     ██║╚██╗██║██║ ██╔██╗ ╚██╗ ██╔╝██║██║╚██╔╝██║"
+      "     ██║ ╚████║██║██╔╝ ██╗ ╚████╔╝ ██║██║ ╚═╝ ██║"
+      "     ╚═╝  ╚═══╝╚═╝╚═╝  ╚═╝  ╚═══╝  ╚═╝╚═╝     ╚═╝"
+    ];
+  };
 
-  custom_header_quotes = helpers.defaultNullOpts.mkListOf (with types; listOf str) "[]" ''
-    Example:
-    ```nix
+  custom_header_quotes = helpers.defaultNullOpts.mkListOf' {
+    type = with types; listOf str;
+    pluginDefault = [ ];
+    # description = ""; # TODO
```

Comment:
```suggestion
    # description = ""; # TODO
```
?

Reply:
Previous option's description only contained the example. Need to add a new description.

Reply:
Added part of upstream's description